sentry_streams/sentry_streams/examples/blq_fn.py

```diff
@@ -0,0 +1,35 @@
+import json
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from enum import Enum
+
+
+class DownstreamBranch(Enum):
+    DELAYED = "delayed"
+    RECENT = "recent"
+
+
+@dataclass
+class KafkaMessage:
+    key: str
+    headers: dict[str, str]
+    value: str
+    timestamp: datetime
+
+    @classmethod
+    def unpack(cls, msg: str) -> "KafkaMessage":
```

Comment:
Please add `from __future__ import annotations` then you can use the name of a class declared later.
`def unpack(cls, msg: str) -> KafkaMessage:`